package.json

```diff
@@ -41,8 +41,41 @@
               ".git",
               ".github"
             ]
-          }
+          },
+          "sourcekit-lsp.serverPath": {
```

Comment:
We can probably make his a lot more user friendly by grouping the settings into a submenu and referring to the "Language Server" instead of "SourceKit-LSP". I can take a stab at that if you want.

Reply:
As with code I want to keep these as close to original.